action.yml

```
@@ -308,4 +308,9 @@ runs:
           if [ "${{ runner.os }}" != "Windows" ]; then
             MAYBE_SUDO="sudo -E"
           fi
-          timeout --verbose --kill-after=1s ${TIMEOUT} ${MAYBE_SUDO} tailscale up ${TAGS_ARG} --authkey=${TAILSCALE_AUTHKEY} --hostname=${HOSTNAME} --accept-routes ${ADDITIONAL_ARGS}
+          for i in {1..5}; do
```

Nit: this might be good to expose as an input that defaults to 5 in case end users want / need to tweak this.

Something like:
```
  retries:
    description: 'Number of times to retry the tailscale --up command'
    required: false
    default: '5'
 ```
 and then:
```suggestion
          for i in {1..${{inputs.retries}}}; do
```
 might do it.